articles/architecture-scenarios/b2c.md

```diff
@@ -1,108 +1,144 @@
+Phase 1 consists of a number of workstreams, with a number of topics in each. The workstreams, topics and the order in which you address each is important, so we recommend you follow the guidance prescribed. That’s not to say you can’t or shouldn’t tackle work in parallel: User Provisioning and User Authentication, for example, could be tackled independently and at the same time, and these could both be tackled in parallel with your Branding efforts. In the majority of successful integration cases we’ve also found that different teams tackle different streams, and that this can provide significant benefit: your design and development team(s) would typically tackle implementation whilst at the same time your branding team would tackle Auth0 asset customization thus reducing overall time to market.
 
-Social login also provides consent screens. So if users give consent, your application can access select user profile attributes about the user.
+The diagram below provides an overall view of the planning associated with Phase 1. By Milestone 1, you will have completed major work required to integrate an application, will have addressed the most significant risk items, and will also be able to provide demonstrable functionality to stakeholders too. 
 
-## Single sign-on
+### Architecture
 
-If you offer a suite of applications, you may need [single sign-on (SSO)](/sso/current) across them, so users only have to log in once.
+[Architecture](/architecture-scenarios/implementation/b2c/tenant-architecture) is the first workstream you will cover, with tenant provision being the precursor to all others. Other topics to address at this stage include: 
 
-Auth0 supports integration with applications that externalize authentication using  industry standard identity protocols: OIDC/OAuth, SAML2 or WS-Fed. Once integrated and configured, your connected applications can use other social identity providers, an Auth0 database, or a database that stores user identities. Auth0 serves as the broker between the applications and the different identity providers.
+* Custom domains
+* Tenant association
+* Support for the Software Development Life Cycle (SDLC)
+    
+### User provisioning
 
-Now when a user signs in to one of your applications, they can access other applications integrated with Auth0 without having to log in again. This will be true until their SSO session expires. You should configure the SSO session length within Auth0 to meet security policies.
+[User provisioning](/architecture-scenarios/implementation/b2c/user-provisioning) is the next workstream, and this can be done in parallel with [user authentication](#user-authentication). We've found that the most successful implementations address the following topics at this state during phase 1, however precisely what you tackle will depend on your specific requirements. For example, you may not need user migration, or you may already have an existing sign up mechanism that can be leveraged. 
```

Comment:
```suggestion
[Provisioning](/architecture-scenarios/implementation/b2c/user-provisioning) is the next workstream, and this can be done in parallel with [Authentication](#authentication). We've found that the most successful implementations address the following topics at this state during phase 1, however precisely what you tackle will depend on your specific requirements. For example, you may not need user migration, or you may already have an existing sign up mechanism that can be leveraged. 
```